malos/heartbeat.proto

```diff
@@ -63,3 +64,7 @@ enum Sensors {
   UV = 8;
   ALTITUDE = 9;
 }
+
+enum Integrations {
+  GPIO = 0;
```

Comment:
Something that we noticed recently. We should add a UNDEFINED  enumerator to protos (with value 0) because otherwise their default value is the first enumerator and you cannot tell if the field has been set or not.

In this case it's not really a problem because the field above (integrations) is a repeated field. But it's better to play it safe.